make-gerrit-user.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -xe
+# Env vars expected:
+# CLUSTERNAME, SECRET_FILE, SECRET_KEY, ADMIN_USER, USERNAME
+
+. config
+
+WORKDIR=$(mktemp ./gerrit-resources-XXX)
+GIT_TOPIC="add-$CLUSTERNAME"
+
+cd $WORKDIR
+SECRET_FILE="${SECRET_FILE:-~/keys/jenkins_mcp.pub}"
```

Comment:
I added fallback to the env var, if it doesn't exist. Shall I remove it as well?